spec.bs

```diff
@@ -2948,6 +2948,21 @@ The <dfn for=Navigator method>getInterestGroupAdAuctionData(|configIDL|)</dfn> m
 1. Let |p| be [=a new promise=].
 1. Let |queue| be the result of [=starting a new parallel queue=].
 1. [=parallel queue/enqueue steps|Enqueue the following steps=] to |queue|:
+  1. If |config|'s [=auction data config/coordinator=] is not one of the [=implementation-defined=]

```

Comment:
is there a reason this cannot this be checked in [=parse and verify ad auction config=]?

Reply:
Because we need the coordinator to get the encryption key when we generate the encrypted request here, not when we get the encrypted response in the auction config passed to `runAdAuction()`.